pkg/intcode/intcode_test.go

```diff
@@ -18,14 +18,16 @@
 	}
 
 	for expectedRes, inst := range computeInstructions {
-		test.EqualInt(t, Compute(inst, Multiply), expectedRes)
+		test.EqualInt(t, Compute(inst, Multiply, make(chan int)), expectedRes)
 	}
 }
 
 func TestComputeInput(t *testing.T) {
 	const input = 42
 
-	test.EqualInt(t, Compute([]int{3, 0, 4, 0, 99}, Output, input), input)
+	inputs := make(chan int, 1)
+	inputs <- input
+	test.EqualInt(t, Compute([]int{3, 0, 4, 0, 99}, Output, inputs), input)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '3', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)